docs/cudf/source/user_guide/guide-to-udfs.ipynb

```diff
@@ -2360,7 +2573,7 @@
    "id": "fe7eb68b",
    "metadata": {},
    "source": [
-    "- Only numeric nondecimal scalar types are currently supported as of yet, but strings and structured types are in planning. Attempting to use this API with those types will throw a `TypeError`.\n",
+    "- Only numeric nondecimal scalar types are currently supported as of yet, with support for strings in `Series` and `DataFrame` `apply` subject to the caveats outlined above. Attempting to use this API with unsupported types will throw a `TypeError`.\n",
```

Comment:
```suggestion
    "- UDFs are currently only supported for numeric nondecimal scalar types (full support) and strings in `Series.apply` and `DataFrame.apply` (partial support, subject to the caveats outlined above). Attempting to use this API with unsupported types will raise a `TypeError`.\n",
```

Reply:
I am not sure about the "(full support)" for the numeric types case.

Prefer `raise` over `throw` when talking about exceptions since we're not in C++ land.